src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -62,6 +62,7 @@ internal int CurrentPasteOffset
         private readonly ObservableCollection<NodeModel> nodes;
         private readonly ObservableCollection<NoteModel> notes;
         private readonly ObservableCollection<AnnotationModel> annotations;
+        protected readonly PresetsModel presetsCollection;
```

Comment:
I can't really see why this needs to be `protected readonly`. Can't a subclass simply get access through the property?